src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts

```diff
@@ -307,3 +331,13 @@ export class OpenSheetMusicDisplay {
     }
     //#endregion
 }
+
+export interface OSMDOptions {
```

Comment:
We need proper docstrings for Typedoc here. On the interface itself as well as on the options. The comments as written now won't make it into the Typedoc output.

Reply:
Second that

Reply:
done in next commit